products/keda.md

```diff
@@ -0,0 +1,72 @@
+---
+title: KEDA
+category: server-app
+tags: kubernetes
+permalink: /keda
+releasePolicyLink: https://keda.sh/docs/latest/operate/cluster/
+changelogTemplate: "https://github.com/kedacore/keda/releases/tag/v__LATEST__"
+eolColumn: false
```

Comment:
No EOL information ? Would it be possible to give a rough idea based on the latest releases. 

As an example you can have a look at Kafka (https://endoflife.date/apache-kafka), which does not have a documented policy too.

And from what I can see it looks like only the latest release is supported: there seem to be no update once a new release is out.

Reply:
I updated the description and added a few more releases. Let me know if it's OK for you.